gse-afs-ext-base/src/main/java/com/powsybl/gse/afs/ext/base/ImportedCaseCreator.java

```diff
@@ -201,7 +202,7 @@ private void replaceNode(ProjectFolder folder, Case aCase, Scene scene) {
                         if (!folder.getChild(newName).isPresent()) {
                             buildFile(aCase, folder, newName);
                         } else {
-                            replaceNode(folder, aCase, scene);
+                            throw new AfsException(String.format("A node with the same name Already exists in the folder '%s'", folder.getName()));
```

Comment:
This is not the correct exception. text is empty if the user has entered an empty text or clicked the cancel button.
maybe you can just log something instead, or throw a proper exception describing what happened.